elements/pfe-sass/maps/_general.scss

```diff
@@ -1,26 +1,48 @@
+@import "@patternfly/patternfly/sass-utilities/scss-variables";
+
 $pfe-vars: (
-  container-spacer:              16px,
-  container-padding:             16px,
-  content-spacer:                24px,
 
-  font-size:                     16px,
-  line-height:                   1.5,
+  container-spacer:              pf-font-prem(16px),
+  container-padding:             pf-font-prem(16px),
+  content-spacer:                pf-font-prem(24px),
+  content-spacer--heading--lg:   pf-font-prem(32px),      
+  content-spacer--heading--md:   pf-font-prem(24px),
+  content-spacer--heading--sm:   pf-font-prem(16px),
+
+  font-family: '"Red Hat Text", "RedHatText", "Overpass", Overpass, Arial, sans-serif',
+  font-family--heading: '"Red Hat Display", "RedHatDisplay", "Overpass", Overpass, Arial, sans-serif',
+  font-family--code: '"Overpass Mono", Consolas, Monaco, "Andale Mono", monospace',
+
+  line-height--heading:     1.2,
+  line-height:              1.3,
 
-  font-family:                   '"Overpass", Overpass, Helvetica, helvetica, arial, sans-serif',
-  font-family--heading:          '"Overpass", Overpass, Helvetica, helvetica, arial, sans-serif',
-  font-family--code:             '"Overpass Mono", Consolas, Monaco, 'Andale Mono', monospace',
+  // General font size variables, used with math to set t-shirt variable values
+  root-font-size:           1em,  // must be EM or PX unit, everything else is rems
 
-  font-size--heading--alpha:     32px,
-  font-size--heading--beta:      24px,
-  font-size--heading--gamma:     21px,
-  font-size--heading--delta:     18px,
-  font-size--heading--epsilon:   16px,
-  font-size--heading--zeta:      14px,
+  base--text:               1rem,      // 16px
+  base--heading:            1.25rem,   // 20px
+  base--heading-lg:         1.5625rem, // 25px
 
-  font-weight--light:            300,
+  base--text--mobile:       0.9rem,    // 14.4px
+  base--heading--mobile:    1.0625rem, // 17px
+  base--heading-lg--mobile: 1.25rem,   // 20px
+ 
+  // Global Font sizes as t-shirt sizes (see also the typography maps for heading & text vars).
+  font-size--3xl:           calc(1.92  * var(--pfe-theme--base--heading-lg, #{pf-font-prem(25px)} )), // 48px             
```

Comment:
```suggestion
  font-size--3xl:           calc(1.92  * var(--#{$repo}-theme--base--heading--lg, #{pf-font-prem(25px)} )), // 48px             
```

Reply:
I think we are using the --pfe convention everywhere instead of #{$repo} for the sake of readability?

Reply:
+1 good point

Reply:
Just the BEM then